app/views/layouts/_sidebar.html.erb

```diff
@@ -35,11 +35,31 @@
         <% if current_user.casa_admin? || current_user.supervisor? %>
           <%= render partial: "layouts/other_duties_nav_item" %>
         <% end %>
-        <li class="<%= active_class(casa_cases_path) %> nav-item">
-          <%= link_to casa_cases_path do %>
-            <i class="lni lni-folder mr-10"></i>
-            <%= cases_index_title %>
-          <% end %>
+
+       <% if policy(:application).see_banner_page? %>
```

Comment:
@Rafael-Martins - Sorry, seems like you mistakenly moved banner changes here 😅

Reply:
Yes I did hahaha, Already fixed!



I also createad the partial, made it a dropdown only for volunteer and fixed/added a test to cover the new dropdown!